<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FanViewSettingDlg</class>
 <widget class="QWidget" name="FanViewSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display Setting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 214, 241);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>550</width>
     <height>190</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 214, 241);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>主视图</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>525</width>
       <height>125</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>525</width>
       <height>125</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>颜色设置</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>72</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>颜色类型:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="colorsel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>145</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 225, 225);</string>
      </property>
      <property name="currentText">
       <string>红蓝色系</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>红蓝色系</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>灰色系</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="ribbon">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>310</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 blue,stop:0.25 aqua, stop:0.5 lime,stop:0.75 yellow, stop:1 red);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="colorValue0">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#colorValue0 {
background-color: rgb(0, 0, 255);
}</string>
      </property>
      <property name="text">
       <string>1e+018</string>
      </property>
     </widget>
     <widget class="QPushButton" name="colorValue1">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>70</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>1e+025</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="onOK">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 214, 241);
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="onCancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
